src/main/resources/userEmailTemplate-feedbackSessionUnpublished.html

```diff
@@ -0,0 +1,16 @@
+Hello ${userName},
+
+<p/>
+${instructorFragment}
+
+<p/>
+The feedback responses for the following feedback session is now closed for viewing.
+<br>&nbsp;&nbsp; Course: [${courseId}]${courseName}
+<br>&nbsp;&nbsp; Feedback Session Name: ${feedbackSessionName}
+<br>
+<p/>
+If you encounter any problems when using the system, you can email TEAMMATES support team at ${supportEmail}.
+<p/>
+Regards,
+<br>
+TEAMMATES Team.
```

Comment:
This email content wont do. We need to explain
1. the session has been unpublished. 
2. The result is no longer available for viewing despite the link sent earlier. 
3. The earlier publishing is possibly a mistake. The email sent should be ignored
4. If the session is published again, there will be another email.